app/qml/ChangelogPanel.qml

```diff
@@ -0,0 +1,157 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Layouts
+import QtQuick.Dialogs
+
+import "."  // import InputStyle singleton
+import "./components"
+import lc 1.0
+
+Item {
+  id: root
+
+  signal close
+
+  Keys.onReleased: function( event ) {
+    if (event.key === Qt.Key_Back || event.key === Qt.Key_Escape) {
+      event.accepted = true
+      close()
+    }
+  }
+
+  MessageDialog {
+    id: errorDialog
+
+    title: qsTr( "Getting What's new failed" )
+    buttons: MessageDialog.Ok
+    onButtonClicked: close()
+  }
+
+  Page {
+    width: parent.width
+    height: parent.height
+    anchors.verticalCenter: parent.verticalCenter
+    anchors.horizontalCenter: parent.horizontalCenter
+    clip: true
+
+    background: Rectangle {
+      color: InputStyle.panelBackgroundWhite
+    }
+
+    header: PanelHeader {
+      height: InputStyle.rowHeightHeader
+      width: parent.width
+      color: InputStyle.clrPanelMain
+      rowHeight: InputStyle.rowHeightHeader
+      titleText: qsTr("What's new")
```

Comment:
```suggestion
      titleText: qsTr("Changelog")
```

Reply:
Original design included "What's new", but as we changed the button in settings to "changelog", let's use the same name here too.